### DISCUSSION TITLE
Troubleshooting Malt install

### DISCUSSION THREAD

**Discussion by mantoetoeman** [bnpr/Malt#447] [Q&A] [answered]
I installed a clean version of Blender 3.3.1, downloaded the latest Malt release.
When activating the Malt renderer it hangs instead of opening the python window and have to force quit Blender.
IDK if it is a file access issue.  I disabled UAC, the user account has full file access and I tried running Blender as an admin.  Any tips to troubleshoot this would be appreciated.

AMD ryzen 7 5800 CPU 32GB ram and 12GB 3070 GPU Win10

I tried a few search terms to see if there was discusison already, I didn't find one.  I'm sorry if this has been addressed.

**Comment by pragma37** [comment, marked as answer]
Make sure you have followed the install instructions: https://github.com/bnpr/Malt/#install
If you did so and it still doesn't work, please open a bug report with the required info:
https://github.com/bnpr/Malt/issues/new?template=bug_report.yml

**Reply by pragma37** [reply]
Nevermind, I can reproduce the issue myself.
It's an issue with an external dependency, I'll fix it asap.

But for the next time remember to report your bugs in the issues section. 👍

**Reply by pragma37** [reply]
It should be fixed now. 👍

**Reply by mantoetoeman** [reply]
It works now, thank you.  Sorry for pposting in wrong section

**Comment by Quillalex** [comment]
I had the same issues a few days ago. Couldnt even uninstall it without getting a python exception error. Was about to use the bug report form but luckily the devs fixed it before I could do that.
